Classes/Controller/AbstractBackendController.php

```
@@ -1023,7 +1023,9 @@ protected function addTranslationButtons(): void
                 $record['possible_translations'] ??= [];
                 $record['possible_translations'][$languageUid] = $targetUrl;
 
-                if (ExtensionManagementUtility::isLoaded('wv_deepltranslate') && \WebVision\WvDeepltranslate\Utility\DeeplBackendUtility::isDeeplApiKeySet()) {
+                if (ExtensionManagementUtility::isLoaded('deepltranslate_core') &&
+                    \WebVision\Deepltranslate\Core\Utility\DeeplBackendUtility::isDeeplApiKeySet()
```

PHPStan (2.0.3)

Call to static method isDeeplApiKeySet() on an unknown class WebVision\Deepltranslate\Core\Utility\DeeplBackendUtility.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(class.notFound)